nntrainer/layers/layer_context.cpp

```diff
@@ -662,6 +662,10 @@ std::string RunLayerContext::getKernelName(LayerKernel layerKernel) {
     return "dot_cl_fp16";
   case LayerKernel::SGEMM_FP16:
     return "sgemm_cl_fp16";
+  case LayerKernel::SSCAL:
```

Comment:
This cl kernel information should be handled in cl context. Otherwise, we cannot support the custom cl layer. We do need CLLernel Class and it should be managed in Cl_Context. and if the layer is created, then N Cl kernel should be mapped to the layer object. and also, if there is a custom cl layer, the cl kernel must be registered to the cl context. I guess the basic cl kernels for dot, sgemm, etc. could be managed as an basic cl math kernels we provide and CLKernel has the list of them.

Reply:
As discussed, this will be done in future PRs.